src/pages/Contributing/GitConventions/PRConvention/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Pull request convention'
+order: 3
+---
+
+## Naming
+
+The pull request naming follow the same rules as the header in a commit message that you can consult [here](/Contributing/GitConventions/CommitMessages).
+
+* **Here a reminder :** 
+
+    The pull request name is composed of type, scope and subject. **Only the scope is optional**
+
+    ```shell
+    <type>(<scope>): <subject>
+    ```
+
+    ### Type
+
+    The type must contains at least one of those : **build**, **ci**, **docs**, **feat**, **fix**, **perf**, **refactor**, **style**, **test**
+
+    ### Scope (optionnal)
+
+    The scope should precise the npm package you worked on : **tools**, **designer-kit**, **icons**, **styles**, **tokens**, **web-fonts**
+
+    ### Subject
+
+    The subject contains a succinct description of the change
+
+
+## The pull request commit(s)
+
+At the end, your pull request should contains only one commit.
+
+![one commit](commit.PNG)
+
+You can manage your commits using 2 methods :
+* Rebase your commit using the rebase interactive
+* Always amend your commit before push to keep the first one
+
+See details in [Git documentation](https://git-scm.com/book/en/v2).
+
+### Exception
+
+Need to clarify with the core TEAM
```

Comment:
If for some reason you worked on multiple types, scopes, or tasks in the same pull request and conserving multiple commits to be documented in the changelogs and release informations seems the best options, please inform your reviewer so the commits are not squashed before being merged. 

example : 

```
feat(tools/postcss): add a plugin that does x y
feat(pattern/buttons): create a feature that do do x y  
```

In that case the feature in the button require the feature on the dev tools, and both informations should be kept in the history, and for information purpose in the changelogs and release infos. You should specifically inform the reviewer that both commit should be kept when merging into master.